lib/Models/SenapsLocationsCatalogItem.ts

```diff
@@ -0,0 +1,279 @@
+import i18next from "i18next";
+import { computed, runInAction } from "mobx";
+import URI from "urijs";
+
+import Cartesian3 from "terriajs-cesium/Source/Core/Cartesian3";
+import Cartographic from "terriajs-cesium/Source/Core/Cartographic";
+import Ellipsoid from "terriajs-cesium/Source/Core/Ellipsoid";
+import JulianDate from "terriajs-cesium/Source/Core/JulianDate";
+import Property from "terriajs-cesium/Source/Core/Property";
+import sampleTerrain from "terriajs-cesium/Source/Core/sampleTerrain";
+import isDefined from "../Core/isDefined";
+import loadJson from "../Core/loadJson";
+import TerriaError from "../Core/TerriaError";
+import AsyncMappableMixin from "../ModelMixins/AsyncMappableMixin";
+import CatalogMemberMixin from "../ModelMixins/CatalogMemberMixin";
+import SenapsLocationsCatalogItemTraits from "../Traits/SenapsLocationsCatalogItemTraits";
+import { FeatureInfoTemplateTraits } from "../Traits/FeatureInfoTraits";
+import CreateModel from "./CreateModel";
+import GeoJsonCatalogItem from "./GeoJsonCatalogItem";
+import Terria from "./Terria";
+import StratumOrder from "./StratumOrder";
+import LoadableStratum from "./LoadableStratum";
+import { BaseModel } from "./Model";
+import { JsonObject } from "../Core/Json";
+import proxyCatalogItemUrl from "./proxyCatalogItemUrl";
+import createStratumInstance from "./createStratumInstance";
+
+export interface SenapsFeature {
+  type: string;
+  properties: {
+    id: string;
+    description: string;
+    endpoint: string;
+    hasStreams: boolean | null;
+    streamIds: string[];
+    streamId: string | null;
```

Comment:
Out of curiosity, why is there streamId and streamIds? If there's only one streamId, shouldn't it just be streamIds[0]? Is it a weird Senaps thing?

Reply:
I use `streamId` for the default chart shown in the popup which isn't ideal, I'll relook at it because perhaps with my refactoring I can show multiple streams on my chart now...

Reply:
Have culled streamId - no longer required